### PR TITLE
docs(builder): adds a skeleton project for @theforeman/builder

### PR DESCRIPTION
**Pull request by sharvit**
## Updated Packages

* [ ] root
* [x] @theforeman/builder
* [ ] @theforeman/env
* [ ] @theforeman/vendor
* [ ] @theforeman/vendor-dev
* [ ] @theforeman/vendor-core

## PR Type

* [ ] Bugfix
* [x] Feature
* [ ] Code style update (whitespace, formatting, missing semicolons, etc.)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] Other… Please describe:

## Description

adds a skeleton project for `@theforeman/builder`
the main content here is documentations and dummy scripts that will explain the general design of this package

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

## Does this PR fixes open issues?

* [ ] Yes
* [x] No

## Do you use this PR in another repository?

* [ ] Yes
* [x] No

### CODE REVIEW

File: packages/builder/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+> Build production and development bundle files for foreman core and plugins.
+
+[![Package Version](https://img.shields.io/npm/v/@theforeman/builder.svg?style=flat-square)](https://www.npmjs.com/package/@theforeman/builder)
+[![Downloads Status](https://img.shields.io/npm/dm/@theforeman/builder.svg?style=flat-square)](https://npm-stat.com/charts.html?package=@theforeman/builder&from=2016-04-01)
+[![Build Status: Linux](https://img.shields.io/travis/theforeman/foreman-js/master.svg?style=flat-square)](https://travis-ci.org/theforeman/foreman-js)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+
+# **This package dosen't contain any implementation yet and only act as a skeleton and as the general design**
```
Comment (xprazak2) [inline packages/builder/README.md]: s /dosen't/doesn't/

File: packages/builder/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@theforeman/builder",
+  "version": "3.1.1",
+  "description": "Build production and development bundle files for foreman core and plugins.",
+  "author": "Avi Sharvit <sharvita@gmail.com>",
+  "homepage": "https://github.com/sharvit/foreman-js#readme",
```
Comment (xprazak2) [inline packages/builder/package.json]: https://github.com/theforeman/foreman-js#readme

File: packages/builder/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "bugs": {
+    "url": "https://github.com/sharvit/foreman-js/issues"
```
Comment (xprazak2) [inline packages/builder/package.json]: Same as above

File: packages/builder/package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/sharvit/foreman-js.git",
```
Comment (xprazak2) [inline packages/builder/package.json]: Same as above

**Comment by xprazak2** [comment]
Nice.

**Comment by sharvit** [comment]
Thanks @xprazak2, updated

**Comment by xprazak2** [comment]
LGTM, will need a rebase though.

**Comment by sharvit** [comment]
Thanks @xprazak2, rebased

**Pull request merged**